kmeta/names.go

```diff
@@ -58,6 +59,7 @@ func ChildName(parent, suffix string) string {
 			// remove it.
 			return strings.TrimRight(ret, "-")
 		}
+		// nolint:gosec // No strong cryptography needed.
```

Comment:
Other than cross-version compatibility, this could be a crew, which is a lot cheaper to compute (for example, it doesn't need to be fixed-duration).